src/fidesops/ops/core/config.py

```diff
@@ -167,12 +174,14 @@ class FidesopsConfig(FidesSettings):
     database: FidesopsDatabaseSettings
     redis: RedisSettings
     security: FidesopsSecuritySettings
-    execution: ExecutionSettings
-    root_user: RootUserSettings
-    admin_ui: AdminUiSettings
-    notifications: FidesopsNotificationSettings
-
-    port: int
+    execution: Optional[ExecutionSettings] = ExecutionSettings()
+    root_user: Optional[RootUserSettings] = RootUserSettings()
+    admin_ui: Optional[AdminUiSettings] = AdminUiSettings()
+    notifications: Optional[
+        FidesopsNotificationSettings
+    ] = FidesopsNotificationSettings()
+
+    port: int = 8080  # Run the webserver on port 8080 by default
```

Comment:
This variable can't be set via an ENV option right now, since there isn't a env var prefix set for the top-level config object

Reply:
Good spot!